components/registry-facade/cmd/setup.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+)
+
+var (
+	hostname string
+	hostfs   string
+	port     int
+)
+
+var setupCmd = &cobra.Command{
+	Use:   "setup",
+	Short: "Updates the /etc/hosts file, updates the CA certificates and creates the registry host for containerd",
+	Run: func(cmd *cobra.Command, args []string) {
+		{
+			log.Info("Creating containerd registry directory...")
```

Comment:
This creates the required file to avoid TLS issues https://github.com/containerd/containerd/blob/main/docs/hosts.md